addon-test-support/@ember/test-helpers/dom/click.js

```diff
@@ -59,7 +59,10 @@ export default function click(target) {
       throw new Error(`Element not found when calling \`click('${target}')\`.`);
     }
 
-    __click__(element);
+    if (!element.hasAttribute('disabled')) {
```

Comment:
I think it should check the property instead of the attribute 🤔

Reply:
updated